dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/model/ProviderModel.java

```diff
@@ -74,6 +74,11 @@ public ProviderMethodModel getMethodModel(String methodName, String[] argTypes)
         return null;
     }
 
+    public List<ProviderMethodModel> getMethodModelList(String methodName) {
+        return methods.get(methodName);
```

Comment:
This may return null value, are you expecting that? Typically when you want to return a List, user will expect an empty list rather than null.